zigbee/Tutorials-and-Examples/Zigbee-Signing-OTA-Image.md

```diff
@@ -0,0 +1,238 @@
+<details>
+<summary><font size=5>Table of Contents</font> </summary>  
+
+- [Signing OTA Image](#signing-ota-image)
+  - [1. Summary](#1-summary)
+  - [2. Gecko SDK version](#2-gecko-sdk-version)
+  - [3. Hardware Required](#3-hardware-required)
+  - [4. Connections Required](#4-connections-required)
+  - [5. Setup](#5-setup)
+    - [5.1. Signing Tool](#51-signing-tool)
+    - [5.2. Verify the OTA file](#52-verify-the-ota-file)
+  - [6. How It Works](#6-how-it-works)
+  - [7. .sls Projects Used](#7-sls-projects-used)
+  - [8. How to Port to Another Part](#8-how-to-port-to-another-part)
+  - [9. Special Notes](#9-special-notes)
+</details>
+
+********
+
+# Signing OTA Image
+
+## 1. Summary ##
+Signing the OTA file is a mandatory requirement of Zigbee Smart Energy applications. For non-SE applications, it's optional. In this page, we will introduce the approach of signing the OTA image of a general Zigbee 3.0 application and verifying it on the client side.
+
+OTA image format is described in section 11.4 of ZCL spec. We will add the signature as a tag.
+<div align="center">
+  <img src="files/ZB-Zigbee-Signing-OTA-Image/ota-file-format.png">
```

Comment:
The image is not very clear to me. The whole image size makes sense. But the header size point to header field. And the arrow point to the signature is not clear what it refer to.